zrpc/internal/serverinterceptors/timeoutinterceptor.go

```diff
@@ -13,18 +13,20 @@ import (
 	"google.golang.org/grpc/status"
 )
 
-var methodTimeout sync.Map
-
-// SetTimeoutForFullMethod set the specified timeout for given method.
-func SetTimeoutForFullMethod(fullMethod string, timeout time.Duration) {
-	methodTimeout.Store(fullMethod, timeout)
+// ServerSpecifiedTimeoutConf defines specified timeout for gRPC method.
+type ServerSpecifiedTimeoutConf struct {
+	FullMethod string
+	Timeout    time.Duration
 }
 
+type specifiedTimeoutCache map[string]time.Duration
```

Comment:
please group the type declarations, like:
```go
type (

)
```

Reply:
done